be/db/migrate/20180206202708_create_events.rb

```diff
@@ -0,0 +1,16 @@
+class CreateEvents < ActiveRecord::Migration[5.1]
+  def change
+    create_table :events do |t|
+      t.datetime :start
+      t.datetime :end
+      t.string :event_type
+      t.string :title
+      t.string :description
+      t.float :latitude
+      t.float :longitude
+      t.datetime :age_start
```

Comment:
What are possible values for age? Should it be a number of ages or months? Datetime looks incorrect to me here now. Let’s change it to integer with a new migration next time I’m around